docs/api/product_variants.rst

```diff
@@ -134,36 +134,13 @@ Example Response
                 {
                     "id": 1,
                     "code": "319bf720-e91d-36f5-aa8e-8c674e3861fb-variant-0",
-                    "option_values": [
```

Comment:
It is just the matter of product serialization. Right now, I just adjusted tests to the current implementation. It is not a problem to bring them back, but I will have to adjust the docs also.